go.mod

```diff
@@ -1,14 +1,14 @@
-module github.com/ONSdigital/dp-api-clients-go
+module github.com/ONSdigital/dp-api-clients-go/v2
```

Comment:
I'm not sure this is right. I think this will rename the module which would break it for everything using it and would also make the original v1 iterations unavailable.

I think the modules approach is to create a new `v2` package within the lib and then put the new v2 functionality in there. Users then update their import to use `/v2`.

Reply:
I can deffo do this approach if needed, however, this hasn't been the way DP has been going to v2 of packages in the past (examples in PR, dp-kafka and dp-log). I think the idea is that we don't support previous versions of a package and people should bump their dependencies to the latest major version

Reply:
There is an official Go modules guide here which outlines the standard  and how to solves this problem I just want to check we are following this pattern, I think we are mostly but not completely.

https://blog.golang.org/v2-go-modules